apps/essimporter/importer.cpp

```diff
@@ -51,6 +51,7 @@ namespace
         {
             for (int x=0; x<128; ++x)
             {
+                assert(image->data(x,y)+2);
```

Comment:
Good idea, but you can probably drop the `+2` from that assert.

[Here's the documentation for osg::Image](http://trac.openscenegraph.org/documentation/OpenSceneGraphReferenceDocs/a00382.html#a9a1861cdedf0d8edc2d1e96f55cae837).

Reading the code it's copying the R,G,B values from the file into the image.  I'd recommend using [setColor](http://trac.openscenegraph.org/documentation/OpenSceneGraphReferenceDocs/a00382.html#ab08a188ade29125510cb3d1cedd8c0a0) instead.

Something like this might work instead:

```
image->setColor({*it++,*it++,*it++,0},{x,y});
```